src/Server/WPHelper.php

```diff
@@ -39,10 +45,12 @@ public function parseRequestParams( $method, array $bodyParams, array $queryPara
 		 * graphql-php's built-in persistentQueryLoader).
 		 *
 		 * @param array $data An array containing the pieces of the data of the GraphQL request
+		 * @param array $request_context An array containing the both body and query params
 		 */
-		$parsed_body_params = apply_filters( 'graphql_request_data', $parsed_body_params );
-		if ( ! empty( $parsed_query_params ) ) {
-			$parsed_query_params = apply_filters( 'graphql_request_data', $parsed_query_params );
+		if ( 'GET' === $method ) {
+			$parsed_query_params = apply_filters( 'graphql_request_data', $parsed_query_params, $request_context );
+		} else {
+			$parsed_body_params = apply_filters( 'graphql_request_data', $parsed_body_params, $request_context );
```

Comment:
nice improvement! could we be a little more explicit about how GET and POST are treated differently? i feel like it would help others to be clear about what we expect from `graphql-php` and could help others interested in using this filter. e.g.:

```
if ( 'GET' === $method ) {
	$request_params = apply_filters( 'graphql_request_data', $parsed_query_params, $request_context );
	return parent::parseRequestParams( $method, [], $request_params );
}

$request_params = apply_filters( 'graphql_request_data', $parsed_body_params, $request_context );
return parent::parseRequestParams( $method, $request_params, [] );
```

sending query params on a POST request is totally valid and could be useful to separate core concerns from biz-logic. reading this, i know that for POST request i need to use 'graphql_request_data' to merge in any query params

Reply:
Absolutely.

Reply:
PR rebased.